airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -61,5 +61,7 @@ acceptance_tests:
   spec:
     tests:
       - spec_path: integration_tests/spec.json
+        backward_compatibility_tests_config:
```

Comment:
why is this change backwards incompatible?

Reply:
because we change spec file and add pattern and format to start date